modules/cnf-installing-amq-interconnect-messaging-bus.adoc

```diff
@@ -15,7 +15,7 @@ To pass PTP fast event notifications between publisher and subscriber on a node,
 
 .Procedure
 
-* Install the AMQ Interconnect Operator to its own `amq-interconnect` namespace. See link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html/deploying_amq_interconnect_on_openshift/installing-router-operator-router-ocp[Installing the AMQ Interconnect Operator].
+* Install the AMQ Interconnect Operator to its own `amq-interconnect` namespace. See link:https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q1/html/deploying_amq_interconnect_on_openshift/adding-operator-router-ocp[Installing the AMQ Interconnect Operator].
```

Comment:
Thank you @aireilly 

**suggestion:**  Maybe we can also update the link text to match the actual page title `Adding the Red Hat Integration - AMQ Interconnect Operator`

LGTM without that suggestion as well. 👍🏼